File: app/src/main/java/ai/elimu/content_provider/ui/emoji/EmojisViewModel.java
package ai.elimu.content_provider.ui.emoji;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class EmojisViewModel extends ViewModel {

    private MutableLiveData<String> text;

    public EmojisViewModel() {
        text = new MutableLiveData<>();
        text.setValue("EmojisViewModel");
    }

    public LiveData<String> getText() {
        return text;
    }
}

File: app/src/main/java/ai/elimu/content_provider/ui/emoji/EmojisViewModel.kt
package ai.elimu.content_provider.ui.emoji

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class EmojisViewModel : ViewModel() {
    private val text = MutableLiveData<String>()

    init {
        text.value = "EmojisViewModel"
    }

    fun getText(): LiveData<String> {
        return text
    }
}